src/rp/util/time.clj

```diff
@@ -10,7 +10,9 @@
 (def patterns
   {:review      "dd-MMM-yyyy"
    :floorplan   "MM/dd/yyyy"
-   :last-update "MM/dd/yyyy HH:mm:ss"})
+   :last-update "MM/dd/yyyy HH:mm:ss"
+   ;; Used for terminate_dm property in Rent Endeca
+   :terminate   "yyyyMMddHHmmss.SSSSSS"})
```

Comment:
Working on refactoring away the use of named formatters.
The only places we make use of them are in a couple of tests.